isthmus/src/main/java/io/substrait/isthmus/SubstraitRelNodeConverter.java

```diff
@@ -3,19 +3,27 @@
 import static io.substrait.isthmus.SqlToSubstrait.EXTENSION_COLLECTION;
 
 import io.substrait.function.SimpleExtension;
+import io.substrait.isthmus.expression.AggregateFunctionConverter;
 import io.substrait.isthmus.expression.ExpressionRexConverter;
 import io.substrait.isthmus.expression.ScalarFunctionConverter;
-import io.substrait.relation.AbstractRelVisitor;
-import io.substrait.relation.Filter;
-import io.substrait.relation.NamedScan;
-import io.substrait.relation.Project;
-import io.substrait.relation.Rel;
+import io.substrait.proto.AggregateFunction;
+import io.substrait.relation.*;
```

Comment:
Minor nit - remove wildcard import. You may want to increase default in IntelliJ which annoyingly defaults to this after three imports from the same package.

Reply:
Removed wildcard import. Thanks for the tips in Intellij setting.